### PR TITLE
zh-translation: sync for some updated p0 doc

### PR DESCRIPTION
**Pull request by ilylia**
ref: https://github.com/servicemesher/istio-official-translation/issues/1714
ref: https://github.com/servicemesher/istio-official-translation/issues/1707
ref: https://github.com/servicemesher/istio-official-translation/issues/1705
ref: https://github.com/servicemesher/istio-official-translation/issues/1704
ref: https://github.com/servicemesher/istio-official-translation/issues/1700
ref: https://github.com/servicemesher/istio-official-translation/issues/1698
ref: https://github.com/servicemesher/istio-official-translation/issues/1695
ref: https://github.com/servicemesher/istio-official-translation/issues/1694
ref: https://github.com/servicemesher/istio-official-translation/issues/1693
ref: https://github.com/servicemesher/istio-official-translation/issues/1685
ref: https://github.com/servicemesher/istio-official-translation/issues/1683
ref: https://github.com/servicemesher/istio-official-translation/issues/1681

### CODE REVIEW

File: content/zh/docs/ops/common-problems/validation/index.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ aliases:
 
 ## 接受无效配置 {#invalid-configuration-is-accepted}
 
-验证 `istio-galley`和`validationwebhookconfiguration` 配置是否存在并且是正确的。 无效的 `apiVersion`、 `apiGroup`和 `resource` 配置应该在两个 `webhook` 其中之一被列举出来。
+验证 `istio-galley validationwebhookconfiguration` 配置是否存在并且是正确的。 无效的 `apiVersion`、 `apiGroup`和 `resource` 配置应该在两个 `webhook` 其中之一被列举出来。
```
Comment (gorda) [inline content/zh/docs/ops/common-problems/validation/index.md]: > `apiVersion`、 `apiGroup`和 `resource`

只是调整一下空格，建议：`apiVersion`、`apiGroup` 和 `resource`

File: content/zh/docs/setup/install/multicluster/shared-vpn/index.md [hunk 2]
```diff
@@ -100,6 +101,7 @@ $ export TELEMETRY_POD_IP=$(kubectl -n istio-system get pod -l istio-mixer-type=
 | `values.global.remoteTelemetryAddress` | 有效的 IP 地址或主机名 | None | 指定 Istio 控制平面的 telemetry Pod IP 地址或远程集群 DNS 可解析的主机名 |
 | `values.sidecarInjectorWebhook.enabled` | true, false | true | 指定是否在远程集群上启用自动 sidecar 注入 |
 | `values.global.remotePilotCreateSvcEndpoint` | true, false | false | 如果设置，将使用 `remotePilotAddress` IP 创建用于 `istio-pilot` 的无选择器的服务和端点，以确保 `istio-pilot.<namespace>` 在远程集群上可通过 DNS 解析。 |
+| `values.global.createRemoteSvcEndpoints` | true, false | false | 如果设置，`istio-pilot`、`istio-telemetry` 和 `istio-policy` 的 selector-less 服务和端点将用相应的远程 IPs：`remotePilotAddress`、`remoteTelmetryAddress` 和 `remotePolicyAddress` 分别创建，这样确保在远程集群中服务名可以通过 DNS 解析。 |
```
Comment (gorda) [inline content/zh/docs/setup/install/multicluster/shared-vpn/index.md]: > IPs

建议：IP

Comment (ilylia) [reply content/zh/docs/setup/install/multicluster/shared-vpn/index.md]: fixed

**Pull request merged**